common/mobilization_options/Babylon_mobilization_option.txt

```diff
@@ -90,30 +90,32 @@ mobilization_option_caplocks = {
 	}
 	on_activate_while_mobilized = {
 		custom_tooltip = {
-			text = mobilization_option_it_hurts_organization_when_you_adjust_equipment_tt
+			text = mobilization_option_armaments_it_hurts_organization_when_you_adjust_equipment_tt
 			add_organization = {
 				value = organization
-				multiply = -0.5
+				multiply = -0.75
 			}
 		}
 	}
 	on_deactivate = {
 		custom_tooltip = {
-			text = mobilization_option_it_hurts_organization_when_you_adjust_equipment_tt
+			text = mobilization_option_armaments_it_hurts_organization_when_you_adjust_equipment_tt
 			add_organization = {
 				value = organization
-				multiply = -0.5
+				multiply = -0.75
 			}
 		}
 	}
 	upkeep_modifier = {
 		goods_input_small_arms_add = 1.5
-		goods_input_ammunition_add = 0.5
+		goods_input_ammunition_add = 1
+		goods_input_small_arms_mult = 0.1
+		goods_input_ammunition_mult = 0.05
```

Comment:
double entries

Reply:
These are mult & add lines, not the same entry